objects/oplayercontroller/Step_0.gml

```diff
@@ -865,14 +865,14 @@ if state != eState.HITSTOP {
 		isJumpingForward = false;
 		jumpHsp = 0;
 		vsp = 0;
-		if !grounded && state != eState.LAUNCHED && state != eState.HURT && state != eState.NEUTRAL_SPECIAL{
+		if !grounded && state != eState.LAUNCHED && state != eState.HURT && state != eState.NEUTRAL_SPECIAL && state != eState.SIDE_SPECIAL{
 			state = eState.IDLE;
 			hsp = 0;
 			grounded = true;
 			frameAdvantage = true;
 			inAttackState = false;
 		}
-		if state == eState.NEUTRAL_SPECIAL {
+		if state == eState.NEUTRAL_SPECIAL || state == eState.SIDE_SPECIAL {

```

Comment:
This does not follow the style guide:
Condition needs to be wrapped in () like if (condition)
Brackets go on a new line, not immediately after the condition